### DISCUSSION TITLE
Enhancing the multi-version documentation layout

### DISCUSSION THREAD

**Discussion by jorgepiloto** [pyansys/actions#155] [Ideas]
With the adoption of the multi-version documentation an issue has been identified. This issue is the lack of a stable URL for the latest stable release. This situation is also detrimental for discovering PyAnsys documentation across the web.

After discussing with @RobPasMue, here is the new structure we would like to propose:

```
.gh-pages
├──version/                                  # A folder collecting all available doc version of a project
│  ├── dev/                                  # The development version
│  ├── stable/                               # The latest stable release
│  ├── .../
│  ├── 0.3/
│  ├── 0.2/
│  ├── 0.1/
│  └── index.html                            # Displays a list of all available versions
├── .nojekyll                                # File required to indicate static website is not of Jekyll type
├── CNAME                                    # File required to indicate the canonical name of the webpage
├── index.html                               # Redirection file to the latest stable version
└── versions.json                            # The file controlling the content of the version switcher
```

Previous structure allows to:

* Always point to the latest stable release: `https://{cname}/version/stable`
* Collect all versions in a single folder instead of having `dev/` and `release/`

The terms `dev` and `stable` are used to avoid confusion between `latest (stable?)` and `stable`.

**Comment by RobPasMue** [comment]
Yes, this is also linked to issue #154 - I totally agree with this approach. The naming can be rethought if you prefer (``dev``, ``stable`` etc) - even the ``version`` folder name. But the structure is clear - we need to handle it like this so that things are more robust, consistent and stable. Thanks for opening the discussion @jorgepiloto - you've got my full support.

**Comment by RobPasMue** [comment]
Pinging @pyansys/pyansys-core team for discussion!

**Reply by germa89** [reply]
I have no opinion about this. From the text, it seems reasonable. If you guys think it is a good idea, I fully support it.

**Comment by clatapie** [comment]
I fully agree with it! It should also make the repository structure more intuitive.

**Comment by MaxJPRey** [comment]
It looks solid.

**Comment by jorgepiloto** [comment]
This got implemented in #163.
